js/app.js

```diff
@@ -117,4 +162,45 @@ const resetForm = () =>{
 }
 
 
-  
\ No newline at end of file
+// delete all stories 
```

Comment:
Write a code that only deletes a row in localstorage. You do not have to remove html tags. Update the list after removing a row.